Wireshark/plugins/v2gmsg.lua

```diff
@@ -10,11 +10,34 @@
 --
 -- See license file (dsV2Gshark_LICENSE.txt)
 --
-DS_V2GSHARK_VERSION = "1.3.0" -- DO NOT CHANGE
+
+-- do OS specific stuff, required to properly load v2g libs
+local plugins_path -- path to the plugins directory of this script
+local lib_pattern
+if package.config:sub(1, 1) == "\\" then
+    -- WINDOWS
+    plugins_path = debug.getinfo(1, "S").source:sub(2):match("(.*[/\\])") or "./"
+    lib_pattern = "?.dll"
+else
+    -- UNIX
+    plugins_path = debug.getinfo(1, "S").source:sub(2):match("(.*/)") or "./"
+    lib_pattern = "?.so"
+end
+local wireshark_path = plugins_path .. "../"
+if not string.find(plugins_path, package.path) then
+    -- extend path (where to load .lua files)
+    package.path = package.path .. ";" .. plugins_path .. "?.lua"
+end
+if not string.find(wireshark_path, package.cpath) then
```

Comment:
Don't you need to check `plugins_path` here too?

Reply:
I think it is sufficient to check`wireshark_path` here, since in this case `plugins_path` is somehow implied